client/src/pages/OfferHelp.js

```diff
@@ -177,7 +177,7 @@ const OfferHelp = withRouter((props) => {
 
   useEffect(() => {
     setTransition(!transition);
-  },[]);
```

Comment:
Also here

Reply:
Yeah this `exhaustive-deps` thing we need to look more into. Either it's something we should ignore completely or we're not using react hooks properly.

Reply:
I think it is the latter more than anything else.